CONTRIBUTING.md

```diff
@@ -1,19 +1,19 @@
-# Contributing to Firefox Reality
+# Contributing to Wolvic
 
-Firefox Reality welcomes contribution from everyone. Here are the guidelines if you are thinking of helping us.
+Wolvic welcomes contribution from everyone. Here are the guidelines if you are thinking of helping us.
 
 
 ## Contributions
 
-Contributions to Firefox Reality or its dependencies should be made in the form of GitHub pull requests. Each pull request will be reviewed by a core contributor(someone with permission to land patches) and either landed in the main tree or given feedback for changes that would be required. All contributions should follow this format, even those from core contributors.
+Contributions to Wolvic or its dependencies should be made in the form of GitHub pull requests. Each pull request will be reviewed by a core contributor(someone with permission to land patches) and either landed in the main tree or given feedback for changes that would be required. All contributions should follow this format, even those from core contributors.
 
 Should you wish to work on an issue, please claim it first by commenting on the GitHub issue that you want to work on it. This prevents duplicate efforts from contributors on the same issue.
 
-Head over to [Good First Bugs](https://github.com/MozillaReality/FirefoxReality/labels/good%20first%20issue) to find good first bugs to start with. 
+Head over to [Good First Issue](https://github.com/Igalia/wolvic/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to find good first bugs to start with. 
 
 ## Pull Request Checklist
 
-- Branch from the [main](https://github.com/mozillareality/firefoxreality/tree/main) branch and, if needed, [rebase](https://help.github.com/en/articles/about-git-rebase) to the current main branch before submitting your pull request. If it doesn't merge cleanly with main, you may be asked to [rebase](https://help.github.com/en/articles/about-git-rebase) your changes.
+- Branch from the [main]([https://github.com/mozillareality/firefoxreality/tree/main](https://github.com/Igalia/wolvic/tree/main)) branch and, if needed, [rebase](https://help.github.com/en/articles/about-git-rebase) to the current main branch before submitting your pull request. If it doesn't merge cleanly with main, you may be asked to [rebase](https://help.github.com/en/articles/about-git-rebase) your changes.
```

Comment:
There is a spurious `[https://github.com/mozillareality/firefoxreality/tree/main]` here.

Reply:
Woops! Good catch.